packages/teleport-shared/src/node-handlers/node-to-html/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { HTMLTemplateSyntax } from './types'
+
+// Vue.js template values
+export const DEFAULT_TEMPLATE_SYNTAX: HTMLTemplateSyntax = {
+  interpolation: (value) => `{{ ${value} }}`,
+  eventBinding: (value) => `@${value}`,
+  valueBinding: (value) => `:${value}`,
+  eventEmmitter: (value) => `this.$emit('${value}')`,
+  conditionalAttr: 'v-if',
+  repeatAttr: 'v-for',
+  repeatIterator: (iteratorName, iteratedCollection, useIndex) => {
+    const iterator = useIndex ? `(${iteratorName}, index)` : iteratorName
+    return `${iterator} in ${iteratedCollection}`
+  },
+}
```

Comment:
Are we considering this is as a default fallback? If we are not sending the mappings from `createHTMLTemplateSyntax` from the base component?